tests/slow/src/test/scala/tests/bazel/BazelLspSuite.scala

```diff
@@ -161,17 +161,17 @@ class BazelLspSuite
       _ = assertNoDiff(
         result.value.linesIterator.take(14).mkString("\n"),
         """|Target
-           |  @//:lib
+           |  @//:hello1
            |
            |Tags
-           |  library
+           |  application
            |
            |Languages
            |  scala
            |
            |Capabilities
            |  Debug <- NOT SUPPORTED
-           |  Run <- NOT SUPPORTED
```

Comment:
Why the change? Does Bazel supports run?

Reply:
Previously the "first" target was a library, now its a binary